firmware/Core/Src/telecommands/telecommand_parser.c

```diff
@@ -153,6 +155,60 @@ uint8_t TCMD_get_suffix_tag_uint64(const char *str, const char *tag_name, uint64
     return 0;
 }
 
+/// @brief Searches for a `str` like `\@tag_name=xxxx`, and sets uint8_t array `xxxx` into `value_dest`.
+/// @param str The string (haystack) to search with for the tag.
+/// @param tag_name The tag to search for, including the '@' and '='.
+/// @param value_dest The destination for the value. `*value_dest` will be set.
```

Comment:
How long must this array be?

Reply:
Not sure if you mean the resulting array, but I added comments in the docstring explaining the needed length of both the input string and the result.